widget.py

```diff
@@ -202,6 +206,9 @@ def udp_receive_socket_data(self):
             message_bytes = data[2:]
             header = packet_spec.parse_packet_header(header_bytes)
             message = packet_spec.parse_packet_message(header, message_bytes)
+            #Actuator state handling
+            if(header.TelemetryPacketSubType == packet_spec.TelemetryPacketSubType.ACT_STATE):
+                self.updateActState(message)
             self.plot_point(header, message)
```

Comment:
I don't think this field exists on the PacketHeader class. Also, can we put this in an if else so we're not always calling plot point.